src/net/sourceforge/kolmafia/persistence/ConsumablesDatabase.java

```diff
@@ -152,6 +152,18 @@ public static void reset() {
     ConsumablesDatabase.reset();
   }
 
+  // Used for testing to replicate static initialization
+  public static void forceRestore() {
```

Comment:
can you make two methods: `clear` and `populateAverage`, and call `populateAverage` in the static init? I think this more clearly shows your intent.

You could have `forceRestore` call `clear`, then `populateAverage` to keep the same external interface.

Reply:
Renamed and repackaged a bit.  I think I got your intent and stopped duplication.